### ISSUE TITLE
Captcha error

### ISSUE THREAD

**Issue by Feripe-a** [hyugogirubato/Kamyroll#3]
I just found out about your app. I can't logging into it, it keeps giving a captcha error. "Invalid captcha". But there's no captcha to be done.

**Comment by hyugogirubato** [comment]
I was able to find a solution with @ Feripe-a on discord but there is indeed a problem with a captcha. I will see what I can do to correct this problem preventing the use of the app for certain with mobile data.

**Comment by Feripe-a** [comment]
Thanks.

**Comment by Feripe-a** [comment]
This might be helpful (?).
![Screenshot_2021-07-12-07-21-01-837_com evo inware](https://user-images.githubusercontent.com/84395172/125272253-04720680-e2e2-11eb-819a-fa6bb7e35750.jpg)
![Screenshot_2021-07-12-07-21-11-731_com evo inware](https://user-images.githubusercontent.com/84395172/125272260-05a33380-e2e2-11eb-9e5a-99eac682648d.jpg)
![Screenshot_2021-07-12-07-21-34-337_com evo inware](https://user-images.githubusercontent.com/84395172/125272265-063bca00-e2e2-11eb-9e75-2013580747d1.jpg)
![Screenshot_2021-07-12-07-21-44-381_com evo inware](https://user-images.githubusercontent.com/84395172/125272267-063bca00-e2e2-11eb-8653-b8378ece7dbe.jpg)
![Screenshot_2021-07-12-07-21-55-637_com evo inware](https://user-images.githubusercontent.com/84395172/125272268-06d46080-e2e2-11eb-9b58-5067ad3a1141.jpg)
![Screenshot_2021-07-12-07-22-01-953_com evo inware](https://user-images.githubusercontent.com/84395172/125272269-076cf700-e2e2-11eb-8b1c-e119562c37b2.jpg)
![Screenshot_2021-07-12-07-22-04-969_com evo inware](https://user-images.githubusercontent.com/84395172/125272271-076cf700-e2e2-11eb-9981-7a7e4776f6d8.jpg)

**Comment by hyugogirubato** [comment]
In a display or crash problem yes, but for network errors it is not necessary, but thanks anyway! I’m going to leave that open until I correct the problem.

**Comment by Feripe-a** [comment]
Update!
![Screenshot_2021-07-12-18-48-33-430_com nashi kamyroll](https://user-images.githubusercontent.com/84395172/125360239-f6ec6900-e341-11eb-95d4-1e04a426a468.jpg)
It has something to do with my internet. I went to a friend's house and used his wifi to connect, it worked. I hope it still be connected when I got home.

> network errors

Is it my internet then? Were you able to replicate the error?
